dev/io.openliberty.reporting/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops

```diff
@@ -0,0 +1,54 @@
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWWKF
+#COMPONENTNAMEFOR CWWKF CVE Reporting
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+# -------------------------------------------------------------------------------------------------
+#*******************************************************************************
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit:
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+# 
+#-----------------------------------------------------------------------------------------------------------------------------
+# Message prefix block: CWWKF1700 - CWWKF1799
+#-----------------------------------------------------------------------------------------------------------------------------
+# Message prefix block for io.openliberty.reporting: CWWKF1700 - CWWKF1799
+
+CWWKF1700.reporting.is.enabled=CWWKF1700I: The checking for security vulnerabilities is enabled. Information about this Liberty runtime will be analyzed against known vulnerabilities.
+CWWKF1700.reporting.is.enabled.explanation= The checking for security vulnerabilities is currently enabled.
+CWWKF1700.reporting.is.enabled.useraction=To disable the checking for security vulnerabilities, specify <cveReporting enabled="false" /> in the server.xml file.
+
+CWWKF1701.reporting.is.disabled=CWWKF1701I: The checking for security vulnerabilities is disabled.
+CWWKF1701.reporting.is.disabled.explanation=The checking for security vulnerabilities must be enabled to run.
+CWWKF1701.reporting.is.disabled.useraction=To enable the checking for security vulnerabilities, specify <cveReporting enabled="true" /> in the server.xml file.
+
+more.information.message=For more information, you can review {0}
+
+CWWKF1702.cve.found=CWWKF1702W: Based on an assessment of {0}, review the following security bulletins:
+CWWKF1702.cve.found.explanation=Security vulnerabilities were found.
+CWWKF1702.cve.found.useraction=Review the CVEs in the message and resolve the security vulnerabilities.
+
+CWWKF1703.cve.not.found=CWWKF1703I: Based on analysis of the Runtime, no security vulnerabilities requiring further analysis were found.
+CWWKF1703.cve.not.found.explanation=No security vulnerabilities were found at this time.
+CWWKF1703.cve.not.found.useraction=The following message has no issues.
```

Comment:
Remove "following"

Reply:
Removed the whole useraction as it is not required.

Reply:
I thought all messages needed a user action even if it was `None`.

Reply:
Did a quick search and couldn't find a message without a useraction so I went ahead and added it back in with `None.`